patches/ui-base-BUILD.gn.patch

```diff
@@ -1,12 +1,12 @@
 diff --git a/ui/base/BUILD.gn b/ui/base/BUILD.gn
-index 085ef60055790df6a98f6b6c4b567e6dbf7b9785..03003fb05493fe5c852f4876be98ac6a8576b8eb 100644
+index 085ef60055790df6a98f6b6c4b567e6dbf7b9785..a4741f5d5a7af098da8f99a1916f2b5f948799e0 100644
 --- a/ui/base/BUILD.gn
 +++ b/ui/base/BUILD.gn
 @@ -408,6 +408,7 @@ jumbo_component("base") {
      "//base:base_static",
      "//base:i18n",
      "//base/third_party/dynamic_annotations",
-+    "//brave/ui/webui/resources",
++    "//brave/ui/base:chromium_deps",
```

Comment:
this is good, but I think we should do a `deps += [ "//brave..." ]` to avoid conflicts

Reply:
done